float32[3] reference_color
sensor_msgs/Image input
---
sensor_msgs/Image output
